pom.xml

```diff
@@ -165,6 +165,7 @@
         <groupId>org.eclipse.jetty</groupId>
         <artifactId>jetty-maven-plugin</artifactId>
         <version>9.3.0.M2</version>
+        <inherited>false</inherited>
```

Comment:
This is unrelated, but gets rid of errors when running jetty:run from the subprojects directly because it would pick up the parent configuration.